algoliasearch_django/version.py

```diff
@@ -1 +1 @@
-VERSION = '1.7.3'
+VERSION = '1.7.4'
```

Comment:
I believe that as you won't be able anymore to use the underlying python client the same way you have to make it a new major

Reply:
Very good point!

```suggestion
VERSION = '2.0.0'
```